src/main/kotlin/xyz/crossward/service/StatsService.kt

```diff
@@ -0,0 +1,35 @@
+package xyz.crossward.service
+
+import org.springframework.stereotype.Service
+import xyz.crossward.repository.EntryRepository
+
+@Service
+class StatsService(
+    private val entryRepository: EntryRepository
+) {
+    fun getAverageTimeByUserId(userId: String): Double {
```

Comment:
Code looks great, but it can be simplified if we use the built in average function
```Kotlin
fun getAverageTimeByUserId(userId: String): Double =
    entryRepository.findTimesByUserId(userId).average()
```